Various/pandabot_ChordGun.lua

```diff
@@ -2,10 +2,8 @@
 Description: ChordGun
 Author: pandabot
 License: MIT
-Version: 1.8
-Changelog: + now preserving existing note lengths if notes are selected
-+ added ability to move midi editor cursor with left and right arrow buttons
-+ added persistence for floating window position
+Version: 1.13
+Changelog: + added code to make ChordGun window always on top but left it commented out, requires julian sader extension and only works on Windows
```

Comment:
This PR's commits will be squashed when merging. The intermediate versions won't be indexed. This changelog should be complete compared to 1.8.